ctru-rs/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![crate_name = "ctru"]
 #![feature(test)]
 #![feature(custom_test_frameworks)]
+#![feature(try_trait_v2)]
```

Comment:
I find the use of the new `Try` trait very elegant (especially compared to the use of a macro, which was my initially scrapped idea). Even so, I must say I don't really enjoy the use of even more unstable features, but this will probably be stabilized soon so it shouldn't be much of a problem.

Reply:
I agree that we should be careful about adding new unstable features to depend on. Hopefully this one stabilizes before too long, but it would be nice to eventually build on stable.

It isn't written like this currently, but I think we could use `#[cfg_attr(test)]` for the other features here. `pthread-3ds` has some other unstable features too I guess, but it would be nice if we can reduce the scope to avoid becoming perma-unstable IMO.